circuit_puzzles/programs/announcer_mutate.clsp

```diff
@@ -34,21 +34,21 @@
               (sha256 ONE LAUNCHER_ID)
               (sha256 ONE (if new_puzzle_hash new_puzzle_hash INNER_PUZZLE_HASH))
               (sha256 ONE APPROVED)
-              (sha256 ONE DEPOSIT)
+              (sha256 ONE deposit)
               (sha256 ONE DELAY)
               (sha256 ONE atom_value)
               (sha256 ONE COOLDOWN_START)
               (sha256 ONE LAST_PENALTY_INTERVAL)
               (sha256 ONE (+ current_timestamp DELAY))
             )
-            DEPOSIT
+            deposit
```

Comment:
needs a check that deposit >= DEPOSIT, o/w mutate operation can be used to withdraw from bond

Reply:
It's fine if they cheat, because they'll get penalized if they do this. No need to curry extra variables

Reply:
what if the entire deposit gets withdrawn? there'd be nothing left to penalise. would effectively be a way to skip the cooldown period